content/docs/sidebar.json

```diff
@@ -37,21 +37,12 @@
   {
     "slug": "start",
     "label": "Get Started",
-    "source": "start/index.md",
     "children": [
       {
         "slug": "data-management",
-        "label": "Data Management",
-        "source": "data-management/index.md",
         "children": [
-          {
-            "label": "Data and Model Access",
-            "slug": "access"
-          },
-          {
-            "label": "Data Pipelines",
-            "slug": "pipelines"
-          },
+          "data-and-model-access",
+          "data-pipelines",
```

Comment:
- Significantly simplifies this file
- Uses more complete URLs (good for SEO I think)

Reply:
Removing `source` from things might break the "Edit on GitHub" links unless we update the sidebar logic to better accept them. Might not, but we need to make sure to check.

Reply:
Hmmm thanks for letting me know @rogermparent. I did not know that (probably something we can improve on in the engine...) I'll reinstate the `source`s for now ⏳

Reply:
> unless we update the sidebar logic to better accept them

https://github.com/iterative/gatsby-theme-iterative/issues/77

> Might not, but we need to make sure to check

P.s. I did check and the button links were broken indeed (in the review app).

Reply:
The button on the review app were broken because the link generated based on `master/main` tree. I checked by changing the path to current branch tree and it's working. 

The logic is intact to link the file source using a simple string slug. And, the `source` field is also getting populated accordingly.
<img width="1353" alt="Screen Shot 2022-09-14 at 15 32 52" src="https://user-images.githubusercontent.com/20840228/190122134-dae28cc5-8018-4c23-a7b4-74e38a665384.png">

Reply:
That would be the case for some of them, but the result of the logic on some nested items, for example `docs/start`, render the link as `https://github.com/iterative/dvc.org/blob/main/content/docs/start.md` where the file is actually at `https://github.com/iterative/dvc.org/blob/main/content/docs/start/index.md`. We can likely just make assume the file is as `index.md` and that'd be all we need to support 90+% of source-less cases.

Reply:
Well I already rolled this back for now. Feel free to take it to https://github.com/iterative/gatsby-theme-iterative/issues/77 😬

Reply:
Okay, I checked on the rolled-back version of the preview app. 

Yes, we do need to define the source for items with `index.md`.